common.mk

```diff
@@ -582,26 +588,22 @@ optee-os-devkit:
 # fTPM Rules
 ################################################################################
 
-# The fTPM implementation is based on ARM32 architecture whereas the rest of the
-# system is built to run on 64-bit mode (COMPILE_S_USER = 64). Therefore set
-# TA_DEV_KIT_DIR manually to the arm32 OPTEE toolkit rather than relying on
-# OPTEE_OS_TA_DEV_KIT_DIR variable.
 FTPM_FLAGS ?= 						\
-	TA_CPU=cortex-a9				\
-	TA_CROSS_COMPILE=$(AARCH32_CROSS_COMPILE)	\
-	TA_DEV_KIT_DIR=$(OPTEE_OS_PATH)/out/arm/export-ta_arm32 \
-	CFG_TA_DEBUG=y CFG_TEE_TA_LOG_LEVEL=4 CFG_TA_MEASURED_BOOT=y
+	CROSS_COMPILE=$(CROSS_COMPILE_S_USER)	\
+	TA_DEV_KIT_DIR=$(OPTEE_OS_TA_DEV_KIT_DIR) \
+	CFG_MS_TPM_20_REF=$(MS_TPM_20_REF_PATH) \
+	O=out
```

Comment:
As per current https://github.com/OP-TEE/optee_ftpm/blob/master/README.md, `CFG_TA_MEASURED_BOOT` is default disabled. I think it should be enabled here, as previously done. 

Should `CFG_TA_DEBUG` be set when `DEBUG!=0`?

Reply:
> As per current https://github.com/OP-TEE/optee_ftpm/blob/master/README.md, `CFG_TA_MEASURED_BOOT` is default disabled. I think it should be enabled here, as previously done.

OK

> 
> Should `CFG_TA_DEBUG` be set when `DEBUG!=0`?

Sure, that makes sense.